api/v1beta1/cinder_types.go

```diff
@@ -64,6 +64,10 @@ type CinderSpec struct {
 	// PasswordSelectors - Selectors to identify the DB and ServiceUser password from the Secret
 	PasswordSelectors PasswordSelector `json:"passwordSelectors,omitempty"`
 
+        // +kubebuilder:validation:Optional
+        // NodeSelector to target subset of worker nodes running Cinder services
+        NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
Wrong indentation level.

Also, not sure we should only have the node selector at the top level, because volume and backup services are in the data path (eg: their nodes may need FC HBAs) that we may want to have them on separated nodes from API and scheduler that are hardware independent.